README.md

```diff
@@ -8,6 +8,38 @@ First, obtain an App ID and an API key from https://docs.traveltime.com/api/over
 
 You will need to email sales@traveltime.com or book a demo at https://traveltime.com/book-demo to gain access to TimeFilterFast(Proto).
 
+### Running K6 Tests Locally
+
+Install [K6](https://k6.io/docs/get-started/installation/)
+
+#### time-map
+
+```bash
+k6 run 
+    -e APP_ID={APP_ID}
+    -e API_KEY={API_KEY}
+    -e HOST=api-dev.traveltimeapp.com //optional
+    -e COUNTRY=gb //optional
+    -e TRANSPORTATION='driving+ferry' //optional
+    -e TRAVEL_TIME=7200 //optional
+    scripts/time-map.js
+```
+
+#### time-filter
+
+```bash
+k6 run 
+    -e APP_ID={APP_ID}
+    -e API_KEY={API_KEY}
+    -e HOST=api-dev.traveltimeapp.com //optional
+    -e COUNTRY=gb //optional
+    -e TRANSPORTATION='driving+ferry' //optional
+    -e TRAVEL_TIME=7200 //optional
+    -e DESTINATIONS="100,150,200" // optional
+    -e RANGE=600 //optional
+    scripts/time-filter.js
+```
+
 ### Running K6 Tests with Docker
```

Comment:
Please add the same instructions how to run time-filter-proto and time-map-fast

Reply:
done